frontend/src/__tests__/cypress/cypress/utils/oc_commands/project.ts

```diff
@@ -0,0 +1,43 @@
+import type { CommandLineResult } from '~/__tests__/cypress/cypress/types';
+
+/**
+ * Create an Openshift Project
+ *
+ * @param projectName Project Name
+ * @param displayName Project Display Name
+ * @returns Result Object of the operation
+ */
+export const createOpenShiftProject = (
+  projectName: string,
+  displayName?: string,
+): Cypress.Chainable<CommandLineResult> => {
+  const finalDisplayName = displayName || projectName;
+  const ocCommand = finalDisplayName
+    ? `oc new-project ${projectName} --display-name='${finalDisplayName}'`
+    : `oc new-project ${projectName}`;
```

Comment:
`finalDisplayName` will always be defined therefore the condition isn't necessary for the `ocCommand`.
Any particular reason to even bother setting the display name to be the same as the project name?

Reply:
Yes, that's a redundancy :S
Fixed in [2f556c2](https://github.com/opendatahub-io/odh-dashboard/pull/3028/commits/2f556c2e54428d515ef4ef2a1dc21a318f8b06e9)